diff --git a/node_modules/vaul/dist/index.js b/node_modules/vaul/dist/index.js
index 3ebb4cc..31cf608 100644
--- a/node_modules/vaul/dist/index.js
+++ b/node_modules/vaul/dist/index.js
@@ -1311,6 +1311,7 @@ function Root({ open: openProp, onOpenChange, children, onDrag: onDragProp, onRe
         }
     }
     return /*#__PURE__*/ React__namespace.default.createElement(DialogPrimitive__namespace.Root, {
+        modal,
         defaultOpen: defaultOpen,
         onOpenChange: (open)=>{
             if (!dismissible && !open) return;
diff --git a/node_modules/vaul/dist/index.mjs b/node_modules/vaul/dist/index.mjs
index d49e23a..108ecc4 100644
--- a/node_modules/vaul/dist/index.mjs
+++ b/node_modules/vaul/dist/index.mjs
@@ -1289,6 +1289,7 @@ function Root({ open: openProp, onOpenChange, children, onDrag: onDragProp, onRe
         }
     }
     return /*#__PURE__*/ React__default.createElement(DialogPrimitive.Root, {
+        modal,
         defaultOpen: defaultOpen,
         onOpenChange: (open)=>{
             if (!dismissible && !open) return;
